config/software/sysstat.rb

```diff
@@ -1,10 +1,10 @@
 name "sysstat"
-default_version "11.1.3"
+default_version "12.4.0"
```

Comment:
This version of `sysstat` cannot be compiled by the version of `gcc` we use in our CI:
https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/41290288
```
gcc -o sar -L/opt/datadog-agent/embedded/lib -I/opt/datadog-agent/embedded/include -Wall -Wstrict-prototypes -pipe -O2 sar.o act_sar.o format_sar.o sa_common.o pr_stats.o librdstats_light.a libsyscom.a -s
Error:
    sa_common.c: In function ‘handle_invalid_sa_file’:
sa_common.c:343:5: warning: implicit declaration of function ‘__builtin_bswap16’ [-Wimplicit-function-declaration]
sa_common.c: In function ‘handle_invalid_sa_file’:
sa_common.c:343:5: warning: implicit declaration of function ‘__builtin_bswap16’ [-Wimplicit-function-declaration]
[…]
ar: creating libsyscom.a
sa_common_sadc.o: In function `handle_invalid_sa_file':
sa_common.c:(.text+0x466): undefined reference to `__builtin_bswap16'
collect2: error: ld returned 1 exit status
make: *** [sadc] Error 1
make: *** Waiting for unfinished jobs....
sa_common.o: In function `handle_invalid_sa_file':
sa_common.c:(.text+0x466): undefined reference to `__builtin_bswap16'
sa_common.o: In function `sa_open_read_magic':
sa_common.c:(.text+0x2078): undefined reference to `__builtin_bswap16'
collect2: error: ld returned 1 exit status
make: *** [sar] Error 1
```

Reply:
DataDog/datadog-agent#6514